packages/components/src/components/carousel-item/carousel-item.test.stories.ts

```diff
@@ -0,0 +1,32 @@
+import '../../solid-components';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+const { argTypes, parameters } = storybookDefaults('sd-carousel-item');
+const { overrideArgs } = storybookHelpers('sd-carousel-item');
+const { generateTemplate } = storybookTemplate('sd-carousel-item');
+
+export default {
+  title: 'Components/sd-carousel-item/Screenshot Tests',
+  tags: ['!autodocs'],
+  component: 'sd-carousel-item',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<div class="slot slot--border slot--text h-16">Default slot</div>`
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters }
+};
+
+/**
+ * This shows sd-carousel-item in its default state.
```

Comment:
I would delete the whole sentence.
The item has no state at all. It is simply a slot.

Reply:
I suggest leaving this because this file is for the screenshot tests and I think it's useful to have some descriptions there. IMO there's no need to review the screenshot tests files because they are a copy of the old documentation.